app/controllers/application_controller.rb

```diff
@@ -7,4 +8,122 @@ class ApplicationController < Sinatra::Base
     set :views, 'app/views'
   end
 
+  enable :sessions
+
```

Comment:
Consider creating helper methods to determine if a user is logged in and to grab user if so.

```
def logged_in?

end

def current_user

end
```